be/src/vec/exec/vaggregation_node.cpp

```diff
@@ -1127,6 +1204,205 @@ Status AggregationNode::_pre_agg_with_serialized_key(doris::vectorized::Block* i
     return Status::OK();
 }
 
+struct PartitionHelper {
+    static constexpr size_t PartitionCountBits = 4;
```

Comment:
set it to session variable, because if the query is very large, 16 sub hashtables will also consume a lot of memory. Maybe spill it to 256 sub hash tables for some query.